lib/Quantization/Base/Base.cpp

```diff
@@ -277,7 +277,10 @@ TensorQuantizationParams chooseQuantizationParams(float min, float max,
     }
   }
 
-  double scale = (max - min) / ((double)qmax - qmin);
+  min = std::max(min, std::numeric_limits<float>::lowest());
```

Comment:
@bertmaher I do not think this is a correct fix.

Having this will make scale values to be way overestimated (std::numeric_limits<float>::lowest() is too small number) and basically accuracy drop will be huge.
This needs to be handled on a caller side, min and max supplied here should not be infinities.
It will allow us to represent non infinity numbers correctly.